neofetch

```diff
@@ -997,6 +997,12 @@ get_distro() {
                     *) distro="GoboLinux $(< /etc/GoboLinuxVersion)"
                 esac
 
+            elif [[ -f /etc/SDE-VERSION ]]; then
+                distro="$(< /etc/SDE-VERSION)"
+                case $distro_shorthand in
+                    on|tiny) distro="${distro% *}" ;;
+                esac
+
```

Comment:
There's no `/etc/os-release` file or `lsb-release` program available? None of the other detection methods work?

Reply:
No, it is not based on another distribution, it did displayed plain Linux. I also added support for the mine packager, and a simple logo.

Reply:
Gotcha. Thanks for clarifying. :+1: